docs/docs/command-line-interface.mdx

```diff
@@ -25,7 +25,7 @@ The command line interface (CLI) gives you easy-to-remember commands for common
 |`rasa x`                |Launch Rasa X locally.                                                                                                                    |
 |`rasa -h`               |Shows all available commands.                                                                                                             |
 
-## Create a new project
+## rasa init
```

Comment:
I changed the headlines as it makes it a lot easier to find and reference the commands. I think it makes more sense from a users perspective as this page is meant to look up commands.

Reply:
I 100% agree!